gremlin-go/driver/graphBinary_test.go

```diff
@@ -51,6 +51,11 @@ func readToValue(buff *bytes.Buffer) interface{} {
 }
 
 func TestGraphBinaryV1(t *testing.T) {
+	var m = map[interface{}]interface{}{
```

Comment:
Just a general question/comment - I believe the empty interface `interface{}` has been recently deprecated in favor of `any`  . At some point we may want to make that change.

Reply:
We are using Go 1.17 since Go 1.18 was released around a month ago. Go 1.18 introduced the any keyword (when they introduced generics).

We discussed upgrading to 1.18, but ultimately decided to stick with 1.17 for the time being for the following reasons:
1. Benefit of upgrade is limited to cleaner code inside our library
2. Users of the library would have to upgrade to Go 1.18 if they were not already on it. This could cause friction in adoption
Obviously open to discussion around upgrading, if you would like to talk more on it.